messybrainz/webserver/views/api.py

```diff
@@ -55,3 +63,111 @@ def get_aka(messybrainz_id):
     (as specified in the clusters table).
     """
     raise NotImplementedError
+
+
+@api_bp.route("/msid", methods=['GET'])
+@crossdomain()
+def get_msid_using_msid():
+    """Returns all MessyBrainz IDs for the MSID in request."""
+
+    msid = request.args.get('id')
+    request_type = request.args.get('request_type')
+    mbid = False
+    includes = request.args.getlist('includes')
```

Comment:
At the summit we agreed to no longer use includes, but resource selectors. Take a look at this document:

https://docs.google.com/document/d/1ClnGKq3rsagERpoAjCjf3Jl1DoCfolzlgj0OXmAy9Pg/edit?usp=sharing

But, I don't see includes being used -- for now just remove them please.